stsynphot/spectrum.py

```diff
@@ -585,11 +585,11 @@ def ebmvx(redlaw_name, ebv):
 
 def load_vega(vegafile=None, **kwargs):
     """Convenience function to load Vega spectrum that is
-    used throughout `stsynphot`.
+    used throughout ``stsynphot``.
 
     Parameters
     ----------
-    vegafile : str or `None`, optional
+    vegafile : str, None
```

Comment:
Why this change?

Reply:
It was complaining that `optional` is not a type. I assume that having `None` as an option is indicative of a optional parameter but we can also add that to the description as well

Reply:
Hmm I just saw this. Lemme see if it fails. If it does, I'll add it back in. Thanks!

Reply:
p.s. I think RTD is green. 🤷